Makefile

```diff
@@ -28,5 +35,22 @@ output/sro_daily-matrix.md: templates/daily_matrix.mail.tmpl.md
              --template  $< \
              --output-file $@
 
+# NTO
+
+nto: output/nto_daily-matrix.html output/nto_daily-matrix.md
+
+output/nto_daily-matrix.html: templates/daily_matrix.tmpl.html
+	go run cmd/main.go --debug daily_matrix \
+	     --config-file examples/nto.yml \
+             --template  $< \
+             --output-file $@
+
+output/nto_daily-matrix.md: templates/daily_matrix.mail.tmpl.md
+	go run cmd/main.go --debug daily_matrix \
+	     --config-file examples/nto.yml \
+             --template  $< \
```

Comment:
`s/--template $</--template $</`

Reply:
there must be some wind in here, every line if further right than the previous one ... I'm not sure how I managed to mess indentation like that!